standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/HiveMetaStoreClient.java

```diff
@@ -327,6 +327,13 @@ private void resolveUris() throws MetaException {
       MetaStoreUtils.logAndThrowMetaException(e);
     }
 
+    if (metastoreUrisString.isEmpty() && "zookeeper".equalsIgnoreCase(serviceDiscoveryMode)) {
+      throw new MetaException("No metastore server available. "
```

Comment:
Please add some additional context here regarding ZooKeeper.  Something like:

`No metastore store service discovered in ZooKeeper`
or
`No metastore store service currently registered in ZookKeeper`

Reply:
done,  @belugabehr thank you very much for the review.